roles/smoker/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: 'Install browser'
+  become: true
+  package:
+    name: '{{ smoker_browser_packages }}'
+
+- name: 'Install Smoker via pytest_project'
```

Comment:
What's the reasoning behind importing separate task files from the role instead of the whole role?

Reply:
Both Robottelo and smoker need to do additional configuration. Technically smoker could drop its variables.json anywhere since it's an absolute path but Robottelo modifies files in the cloned repo. Since I started with Robottelo, I continued this way. And do sort of a callback where I provided a role/task with setup was even worse IMHO. If you know a clean way to do this in Ansible, I'd love to hear it.